docs/use_cases/prometheus-alerts-with-kubescape.md

```diff
@@ -0,0 +1,49 @@
+In the complex environment of a cloud-native enterprise, ensuring the security and stability of Kubernetes clusters is crucial. The following narrative illustrates how a company leverages Kubescape and Prometheus to establish a robust monitoring and alerting framework for their Kubernetes clusters.
+
+
+---
+
+
+#### **Scenario**
+
+An operations team at a leading cloud services provider is focused on bolstering the observability of their Kubernetes clusters. With a variety of services running across different environments, the team prioritizes the setup of Prometheus for metrics collection and alerts for security incidents as identified by Kubescape, allowing for quick and effective incident response.
+
+
+## **Actions**
+
+
+
+1. Prometheus Installation:
+    * The team installs Prometheus using Helm charts, setting up the server, Alertmanager, and node exporters to gather cluster metrics.
+    * They ensure Prometheus has the necessary RBAC permissions for metrics scraping from cluster nodes and pods.
+2. Kubescape Metrics Integration:
+    * Prometheus is then configured to scrape security metrics exposed by Kubescape.
+    * In the absence of native Prometheus-compatible metrics from Kubescape, an intermediary service or exporter is deployed to format Kubescape's output for Prometheus.
+3. Alert Rules Configuration:
+    * The team defines Prometheus alert rules based on security events reported by Kubescape, such as when critical vulnerabilities or specific misconfiguration counts exceed certain thresholds.
+    * Alertmanager is configured to handle these alerts, with capabilities to manage, mute, and inhibit them as needed.
+4. Notification Channels Integration:
+    * Alertmanager is integrated with various notification channels like email, Slack, and OpsGenie.
+    * This setup includes the creation of receiver integrations, routing trees for alerts, and templated messages for clear and actionable alerts.
+5. Alerting System Testing:
+    * The team conducts a simulated security event to verify the alerting process from Kubescape detection to Prometheus alert generation and routing through Alertmanager to the intended channels.
+6. Documentation:
+    * The complete setup, including Prometheus and Kubescape integration, alert rules, notification workflows, and response protocols, is thoroughly documented for clarity and training purposes.
```

Comment:
IMO this whole section needs grounding. Either with a link to how-to documentation or a tutorial.

Reply:
Currently the documentation is located [here](https://hub.armosec.io/docs/prometheus-exporter).
How do we want to incorporate it? Maybe we should add this to the documentation in Kubescape.io? Or because this is a helm integration it shouldn't be in this docs at all?

Reply:
I suggest adding a link to the appropriate documentation or at least a code snippet thatsuggest how to do the setup.

Reply:
Done